docs/porting/libraries.md

```diff
@@ -1,5 +1,224 @@
+
+## Targeting your code for .NET Framework 4.6.1
+
+If your code is not targeting .NET Framework 4.6.1, it's highly recommended that you retarget.  This ensures that you can use API alternatives for cases where .NET Core can't support existing APIs.
+
+For each of your projects in Visual Studio you wish to port, do the following:
+
+1. Right-click on the project and select Properties
+2. In the "Target Framework" dropdown, select ".NET Framework 4.6.1".
+3. Recompile your projects.
+
+And that's it!  Because your projects now target .NET Framework 4.6.1, you can use that version of .NET Framework as your base for porting code.
+
+## Determining the portability of your code
+
+The next step is to run the API Portability Analyzer to generate a portability report that you can begin to analyze.
+
+You'll need to make sure you [understand the API Portability tool](tooling.md) and can generate portability reports for targeting .NET Core.  How you do this will likely vary based on your needs and personal tastes.  What follows are a few different approaches - you may find yourself mixing each approach depending on how your code is structured.
+
+### Dealing primarily with the compiler
+
+This approach may be the best for small projects or projects which don't use many .NET Framework APIs.  The approach is very simple:
+
+1. Run API Port on your project.
+2. Take a quick glance and perhaps use the report as a reference for future errors.
+3. Copy all of your code over into a new .NET Core project.
+4. Work out compiler errors until it compiles, referring to the portability report if needed.
+5. Repeat as needed.
+
+Although this approach is very unstructured, the code-focused approach can lead to resolving any issues quickly, and may be the best approach for smaller projects or libraries.  A project that contains only data models may be an ideal candidate here.
+
+### Staying on .NET Framework until portability issues are resolved
+
+This approach may be the best if you prefer to have code that compiles during the entire process.  The approach is as follows:
+
+1. Run API Port on a project.
+2. Address issues by using different APIs which are portable.
+3. Keep note of any areas where you can't use a direct alternative.
+4. Repeat steps 1-3 for all projects you're porting until you're confident each is ready to be copied over into a .NET Core project.
+5. Copy the code into a new .NET Core projects.
+6. Work out any issues that you've kept note of.
+
+This approach is a bit more careful and structured than simply working out compiler errors, but it is still relatively code-focused and has the benefit of always having code that can compile.  The way you resolve certain issues that couldn't be addressed by just using another API can vary greatly.  You may find that you need to develop a more comprehensive plan for certain projects, which is covered as the next approach.
```

Comment:
Suggested rephrase "This careful approach is more structured than simply working out compiler errors..."